transcribed/page1160.txt

```diff
@@ -0,0 +1,23 @@
+001             ;*SYSTEM* JOBS                                                  PAGE 116
+002
+003     117 016 SCORE:   SKIPL SUPCOR
+004     017 002          PUSHJ P,UPL3
+005     116 017          PUSHJ P,SCORE4
+006     117 013          CLEARM @SYSL
+007     117 013          HRLX A,SYSL
+008     117 013          HRM A,SYSL
+009     001 006          AOS A
+010     001 007          MOVE I A,2000
+011     117 013          ADD S,SYSL
+012     068 004          BLT A,(S)
+013     117 015          MOVEM B,SYSL
+014     117 016 NMCU:    CLEARM SUPCOR
+015     118 003          JRST SCOR
+016
+017     117 015 SCORES:  MOVE S,SYSL
```

Comment:
B,

We can be pretty sure it's not S, because page 1 doesn't define an S accumulator.

Reply:
Label is SCOREX per the table.

Reply:
Aha! I didn't realize how these things are related. Remember, I don't understand the code I'm looking at! :)

So 001 006 though 001 018 defines accumulators (similar to variables?), but 001 021 defines constants I imaginge..?

Reply:
Yes, X=Y defines the symbol X to have the value Y.  A, B, C, etc are conventional names for accumulators, and values 0-17 is the address space for accumulators.  Accumulator definitions are not special in any way, it how they are used that defines them as accumulators, i.e. an instruction `FOO A,X`, A is the accumulator field.